### ISSUE TITLE
High plasma lyso-globotriaosylsphingosine level

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5146]
https://www.ncbi.nlm.nih.gov/pubmed/30875019

**Comment by pnrobinson** [comment]
@Marwaabdelhakim

**Comment by pnrobinson** [comment]
Parent: https://hpo.jax.org/app/browse/term/HP:0003119 (Possibly we need a new subhierarchy).
Label: Elevated circulating lyso-globotriaosylsphingosine concentration
Def; An abnormal increase in the level of globotriaosylsphingosine (Lyso-Gb3) in the blood circulation.
Comment: Fabry disease is an X-linked lysosomal storage disease caused by deficiency of alpha-galactosidase A that affects males and shows disease expression in heterozygotes. Lysosomal accumulation of its substrate globotriaosylceramide (Gb3) that is prominent in endothelial cells is thought to cause the progressive renal insufficiency, cardiac involvement, and CNS pathology in Fabry patients. Plasma Lyso-Gb3 is considered to be a biomarker for Fabry disease. 

PMID:29288396  PMID:30875019

**Comment by pnrobinson** [comment]
adding as HP:0032657
